test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/AbstractService.java

```diff
@@ -293,6 +296,18 @@ private Map<String, Object> getBookkeeperImageSpec(String imageName) {
     }
 
     private Map<String, Object> getResources(String limitsCpu, String limitsMem, String requestsCpu, String requestsMem) {
+        String jsonSrc = "";
```

Comment:
Any reason for this empty string? How are we going to read the configuration from json file ?

Reply:
Added a json file to read the system test config systemTestConfig.json to read the values from. Passed the path of the json file as a parameter SYSTEMTESTPROPERTIES and tried passing it in getsystemTestConfig.